Forge.Forms/src/Forge.Forms/DynamicExpressions/MultiValueConverters/AppendMultiConverter.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Globalization;
+using System.Text;
+using System.Windows.Data;
+
+namespace Forge.Forms.DynamicExpressions.MultiValueConverters
+{
+    public class AppendMultiConverter : IMultiValueConverter
+    {
+        public AppendMultiConverter()
+        {
+        }
+
+        public AppendMultiConverter(object delimeter)
+        {
+            Delimeter = delimeter;
```

Comment:
Not sure if it's a spelling variant but elsewhere in the codebase we use delim**iter**.

Reply:
Just a typo, will fix